tracee/printer.go

```diff
@@ -105,28 +107,45 @@ func newEvent(ctx context, argsNames []string, args []interface{}) (Event, error
 }
 
 type tableEventPrinter struct {
-	tracee  *Tracee
-	out     io.Writer
-	err     io.Writer
-	verbose bool
+	tracee        *Tracee
+	out           io.Writer
+	err           io.Writer
+	verbose       bool
+	containerMode bool
 }
 
 func (p tableEventPrinter) Init() {}
 
 func (p tableEventPrinter) Preamble() {
 	if p.verbose {
-		fmt.Fprintf(p.out, "%-14s %-16s %-12s %-12s %-6s %-16s %-6s %-6s %-6s %-16s %-20s %s", "TIME(s)", "UTS_NAME", "MNT_NS", "PID_NS", "UID", "COMM", "PID", "TID", "PPID", "RET", "EVENT", "ARGS")
+		if p.containerMode {
+			fmt.Fprintf(p.out, "%-14s %-16s %-12s %-12s %-6s %-16s %-11s %-11s %-11s %-16s %-20s %s", "TIME(s)", "UTS_NAME", "MNT_NS", "PID_NS", "UID", "COMM", "PID/host", "TID/host", "PPID/host", "RET", "EVENT", "ARGS")
+		} else {
+			fmt.Fprintf(p.out, "%-14s %-16s %-12s %-12s %-6s %-16s %-5s %-5s %-5s %-16s %-20s %s", "TIME(s)", "UTS_NAME", "MNT_NS", "PID_NS", "UID", "COMM", "PID", "TID", "PPID", "RET", "EVENT", "ARGS")
+		}
 	} else {
-		fmt.Fprintf(p.out, "%-14s %-16s %-6s %-16s %-6s %-6s %-6s %-16s %-20s %s", "TIME(s)", "UTS_NAME", "UID", "COMM", "PID", "TID", "PPID", "RET", "EVENT", "ARGS")
+		if p.containerMode {
+			fmt.Fprintf(p.out, "%-14s %-6s %-16s %-11s %-16s %-20s %s", "TIME(s)", "UID", "COMM", "PID/host", "RET", "EVENT", "ARGS")
+		} else {
+			fmt.Fprintf(p.out, "%-14s %-6s %-16s %-5s %-16s %-20s %s", "TIME(s)", "UID", "COMM", "PID", "RET", "EVENT", "ARGS")
```

Comment:
If removing pids, I suggest keeping tid and not pid, as it's more informative and the process name (comm) already implies that different tids belong to the same process.

Reply:
Don't want to remove pid as it's useful for further analysis. I will keep both tid and pid then